src/Uno.UI/UI/Xaml/Controls/ProgressRing/ProgressRing.wasm.cs

```diff
@@ -1,22 +1,63 @@
 ﻿#if __WASM__
 using System;
-using System.Collections.Generic;
-using System.Text;
-using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Controls.Primitives;
 
 namespace Windows.UI.Xaml.Controls
 {
-    public partial class ProgressRing : Control
+	public partial class ProgressRing : Control
 	{
 		private static void OnForegroundChanged(DependencyObject dependencyObject, DependencyPropertyChangedEventArgs args)
 		{
-			
+
 		}
 
 		private static void OnIsActiveChanged(DependencyObject dependencyObject, DependencyPropertyChangedEventArgs args)
 		{
-		
+			var progressRing = dependencyObject as ProgressRing;
+			var isActive = args.NewValue as bool?;
```

Comment:
Why `bool?` and not `bool`?

For DP callbacks it's generally safe to assume that `dependencyObject` will indeed be a `ProgressRing` and `args.NewValue` will indeed be a `bool`, these are invariants enforced by the DP system.

Reply:
Because these 2 lines are copy-pasted from Android\iOS version where they were like this:
https://github.com/unoplatform/uno/blob/739f7b34d4b1e057b174b342a248749906b603d6/src/Uno.UI/UI/Xaml/Controls/ProgressRing/ProgressRing.Android.cs#L40-L41
So I assumed this was the correct way :)